ocaml/dream/Makefile

```diff
@@ -0,0 +1,8 @@
+.PHONY: deps
+deps:
+	opam switch create . 5.2.0 --yes
+	opam install --deps-only --yes .
+
+.PHONY: serve
+serve:
+	dune exec --root . ./server.exe
```

Comment:
Is this production ready?

Reply:
Afaik, it is production ready in the sense that the server comes with dream is not designed as a development server, and I use dream in production, as well as the official website of [OCaml](https://ocaml.org/), I am not affiliated with OCaml, but found some public-facing code using dream, as in here https://github.com/ocaml/ocaml.org/blob/b34183bcc1e4966390e2240ff920fc2d64cdde09/src/ocamlorg_web/lib/ocamlorg_web.ml#L15